### ISSUE TITLE
getHeight() returns wrong 

### ISSUE THREAD

**Issue by frgithub35** [fdivitto/FabGL#194]
Hi,
Thank you for this great job that's giving me a lot of fun :-) 

I've been playing with Sprites.ino example to know wich resolution is the best with my screen.
so I've changed the line with
`DisplayController.setResolution(VGA_320x200_75Hz);`

with some differents values.
with higher resolution than 320x200, half of the background is not filled in blue.

So, I put
```
  Serial.println(cv.getWidth());
  Serial.println(cv.getHeight());
```
in the setup
And I found that above VGA_640x480_60Hz, getHeight() doesn't return the right number of pixels,
but a lower number of pixel:
examples : 
for 640x480@60Hz
Width = 640
Height = 406

for SVGA_1280x600_60Hz
Width = 1280
Height = 196

Is this a normal behavior ?

Regards

**Comment by fdivitto** [comment]
yes, it is normal behavior when there isn't enough available memory to allocate frame buffer.
VGAController requires one byte per pixel, so 1280x600 would require 1280x600=768KBytes... actually too much for the ESP32, so FabGL can just allocate 196 lines (1280x196=250K).
I suggest you to use paletted controller, like VGA16Controller (one byte contains two pixels) or VGA8Controller.
